artemis/src/main/java/tech/pegasys/artemis/cli/options/LoggingOptions.java

```diff
@@ -28,6 +28,10 @@
   public static final String LOG_DESTINATION_OPTION_NAME = "--log-destination";
   public static final String LOG_FILE_OPTION_NAME = "--log-file";
   public static final String LOG_FILE_NAME_PATTERN_OPTION_NAME = "--log-file-name-pattern";
+  public static final String LOG_WIRE_CIPHER_ENABLED_OPTION_NAME = "--log-wire-cipher-enabled";
+  public static final String LOG_WIRE_PLAIN_ENABLED_OPTION_NAME = "--log-wire-plain-enabled";
+  public static final String LOG_WIRE_MUX_ENABLED_OPTION_NAME = "--log-wire-mux-enabled";
+  public static final String LOG_WIRE_GOSSIP_ENABLED_OPTION_NAME = "--log-wire-gossip-enabled";
```

Comment:
Can we make this `X` prefixed options with the `hidden` flag set?  They're useful to have for debugging but likely to be overly confusing for users and are quite implementation dependent so may change in the future.  The flags in `InteropOptions` are examples of this approach.  They don't show up in the normal help output (we need to make them show up if you run `teku -X` at some point).

Also, sorry I just created a lot of conflicts for you on this..

Reply:
Done
> Also, sorry I just created a lot of conflicts for you on this..

No worries, it merged automatically :)